### DISCUSSION TITLE
Cameras unproject_points

### DISCUSSION THREAD

**Discussion by GuillaumeLeclerc** [facebookresearch/pytorch3d#1642] [Q&A]
Hello,

I have N cameras and N points that I want to unproject. right now it will generate (N, N, 3) points. Is there a way to handle the only compute the diagonal and not more to save resources.

Thanks!

**Comment by bottler** [comment]
Maybe it does what you want if you send the points in as shape (N, 1, 3) instead of (N, 3)?
